modules/map/pnc_map/pnc_map.cc

```diff
@@ -75,11 +75,9 @@ LaneWaypoint PncMap::ToLaneWaypoint(
 double PncMap::LookForwardDistance(double velocity) {
   auto forward_distance = velocity * FLAGS_look_forward_time_sec;
 
-  if (forward_distance > FLAGS_look_forward_short_distance) {
-    return FLAGS_look_forward_long_distance;
-  }
-
-  return FLAGS_look_forward_short_distance;
+  return forward_distance > FLAGS_look_forward_short_distance
+  ? FLAGS_look_forward_long_distance
+  : FLAGS_look_forward_short_distance;
```

Comment:
Use scripts/clang-format.sh to fix coding style please.

Reply:
done